pom/flight_pom.py

```diff
@@ -1,72 +1,55 @@
 
     def station_from_sending_keys(self):
-        reg4 = FlightRegister.variable1_["station_from_sending_keys"]
-        FlightRegister.ObjectGen_.enter_text(self.driver, reg4, values="Chennai")
+        self.ObjectGen_.enter_text(self.driver, self.variable1_["station_from_sending_keys"], values="Chennai")
 
     def station_from_click(self):
-        reg5 = FlightRegister.variable1_["station_from_click"]
-        FlightRegister.ObjectGen_.click_on_element(self.driver, reg5)
+        self.ObjectGen_.click_on_element(self.driver, self.variable1_["station_from_click"])
 
     def station_to_sending_keys(self):
-        reg6 = FlightRegister.variable1_["station_to_sending_keys"]
-        FlightRegister.ObjectGen_.enter_text(self.driver, reg6, values="Mumbai")
+        self.ObjectGen_.enter_text(self.driver, self.variable1_["station_to_sending_keys"], values="Mumbai")
 
     def station_to_click(self):
-        reg7 = FlightRegister.variable1_["station_to_click"]
-        FlightRegister.ObjectGen_.click_on_element(self.driver, reg7)
+        self.ObjectGen_.click_on_element(self.driver, self.variable1_["station_to_click"])
 
     def start_date_select(self):
-        reg8 = FlightRegister.variable1_["start_date_select"]
-        FlightRegister.ObjectGen_.click_on_element(self.driver, reg8)
+        self.ObjectGen_.click_on_element(self.driver, self.variable1_["start_date_select"])
 
     def return_date_click(self):
-        reg9 = FlightRegister.variable1_["return_date_click"]
-        FlightRegister.ObjectGen_.click_on_element(self.driver, reg9)
+       self.ObjectGen_.click_on_element(self.driver, self.variable1_["return_date_click"])
 
     def apirl_month_select(self):
-        reg10 = FlightRegister.variable1_["apirl_month_select"]
-        FlightRegister.ObjectGen_.click_on_element(self.driver, reg10)
+        self.ObjectGen_.click_on_element(self.driver, self.variable1_["apirl_month_select"])
 
     def return_date_select(self):
-        reg11 = FlightRegister.variable1_["return_date_select"]
-        FlightRegister.ObjectGen_.click_on_element(self.driver, reg11)
+        self.ObjectGen_.click_on_element(self.driver, self.variable1_["return_date_select"])
 
     def search(self):
-        reg12 = FlightRegister.variable1_["search"]
-        FlightRegister.ObjectGen_.click_on_element(self.driver, reg12)
-        return FlightRegister.ObjectGen_.validate_element_display(self.driver, FlightRegister.variable1_["book"])
+        self.ObjectGen_.click_on_element(self.driver, self.variable1_["search"])
+        return self.ObjectGen_.validate_element_display(self.driver,self.variable1_["book"])
 
     def book(self):
-        reg13 = FlightRegister.variable1_["book"]
-        FlightRegister.ObjectGen_.click_on_element(self.driver, reg13)
-        return FlightRegister.ObjectGen_.validate_element_display(self.driver, FlightRegister.variable1_["gst_pop_up"])
+        self.ObjectGen_.click_on_element(self.driver, self.variable1_["book"])

```

Comment:
Method name is not clear, I didn't understood what this method does.

Reply:
this is book button click . i have change the name